client/landing/stepper/declarative-flow/flows/site-migration-flow/site-migration-flow.ts

```diff
@@ -57,7 +57,6 @@ const siteMigration: Flow = {
 			STEPS.SITE_MIGRATION_ASSIGN_TRIAL_PLAN,
 			STEPS.SITE_MIGRATION_INSTRUCTIONS,
 			STEPS.ERROR,
-			STEPS.SITE_MIGRATION_ASSISTED_MIGRATION,
```

Comment:
Question: can we remove the step definition entirely in a follow-up PR? It's likely this is the only code using that step.

Reply:
Yes! That was exactly my plan! 😀